Sources/Datadog/RUM/RUMMonitor/RUMCommand.swift

```diff
@@ -298,6 +299,8 @@ internal struct RUMStopUserActionCommand: RUMUserActionCommand {
 internal struct RUMAddUserActionCommand: RUMUserActionCommand {
     var time: Date
     var attributes: [AttributeKey: AttributeValue]
+    var isRage = false
+    var targetView : UIView? = nil // Most likely target for this action
```

Comment:
Is `targetView` necessary? I would prefer not to add dependency to `UIKit` here.

Reply:
@ncreated gave a more detailed comment on this one :)